engines/bastion_katello/app/assets/javascripts/bastion_katello/host-collections/details/host-collection-details.controller.js

```diff
@@ -35,19 +35,26 @@ angular.module('Bastion.host-collections').controller('HostCollectionDetailsCont
             ApiErrorHandler.handleGETRequestErrors(response, $scope);
         });
 
+        $scope.selected = {};
+
         $scope.getHostIds = function() {
             return $scope.selected;
         };
 
+        ContentHostsModalHelper.resolveFunc = $scope.getHostIds;
+
         $scope.host = Host.get({search: "host_collection_id = " + $scope.$stateParams.hostCollectionId}, function (response) {
+            var ids = _.map(response.results, 'id');
```

Comment:
isn't this completely ignoring pagination?  Sending the search ("host_collection_id") to the bulk action would be much preferred.

Reply:
I was just adjusting this to work from before but I can switch it to do that, what is the syntax the call is expecting?

Reply:
Here's the param group description:

https://github.com/Katello/katello/blob/master/app/controllers/katello/api/v2/hosts_bulk_actions_controller.rb#L37-L46

You'd pass {'included': {'search': 'host_collection_id = 3''}}

Reply:
Thanks, that's much easier.